components/automate-ui/src/app/app.module.ts

```diff
@@ -354,7 +358,18 @@ import { WelcomeModalComponent } from './page-components/welcome-modal/welcome-m
     TeamRequests,
     TelemetryService,
     UserPermsRequests,
-    UserRequests
+    UserRequests,
+    // Initilization for warning banner component
+    {
+      provide: APP_INITIALIZER,
+      multi: true,
+      deps: [AppConfigService],
+      useFactory: (appConfigService: AppConfigService) => {
+        return () => {
+          return appConfigService.loadAppConfig();
+        };
+      }
+    }
```

Comment:
What this block does is allow us to call the `AppConfigService`'s `loadConfigApp` before the application renders.  In this case, we are using that time to grab our configuration for the warning banner.